core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/LinkInspectorResourceUtil.java

```diff
@@ -59,6 +55,37 @@ public static void removeResource(String path, ResourceResolver resourceResolver
         }
     }
 
+    public static void createNode(String path, ResourceResolver resourceResolver) {
+        try {
+            Session session = resourceResolver.adaptTo(Session.class);
+            if (session == null) {
+                LOG.warn("Session is null, recreating node is interrupted");
+                return;
+            }
+            JcrUtil.createPath(path, JcrConstants.NT_UNSTRUCTURED, session);
+            session.save();
```

Comment:
We do not specially need `session.save()` to save. We can just call `resourceResolver.commit()`. Therefore, we can do without `adaptTo(session)`, `if(session != null)`, etc., and save some 10 lines of code)